lia-ui/src/main/java/com/lithium/community/android/ui/components/activities/LiCreateMessageActivity.java

```diff
@@ -233,7 +240,8 @@ public void onConfigurationChanged(Configuration newConfig) {
     @Override
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
-        imageSelectionIntent = data;
-        fragment.handleImageSelection(imageSelectionIntent);
+        if(requestCode == LiSDKConstants.PICK_IMAGE_REQUEST) {
+            fragment.handleImageSelection(data);
```

Comment:
Mark `fragment` as `@Nullable` check not `null`.

Reply:
outside onCreate fragment will never be null.